docs/codeql/codeql-cli/about-ql-packs.rst

```diff
@@ -124,7 +124,7 @@ A custom QL pack for queries must include a ``qlpack.yml`` file at
 the pack root, containing ``name``, ``version``,
 and ``libraryPathDependencies`` properties. If the pack contains query suites, you can
 use the ``suites`` property to define their location. Query suites defined 
-here are called "well-known" suites, and can be used on the command line by referring to 
+here are called "known" suites, and can be used on the command line by referring to 
```

Comment:
Here, you're using _known_, but above is _well-known_. The `resolve queries` command also uses _well-known_. It's not a great term, but it is something that has been used for a while.

Reply:
I made the change and then realized that it wasn't a new term. I thought I'd reverted it, but obviously missed this one. Thanks for checking.

Reply:
```suggestion
here are called "well-known" suites, and can be used on the command line by referring to 
```